mira/modeling/gromet_model.py

```diff
@@ -0,0 +1,210 @@
+from typing import List
+from itertools import count
+
+import pytz
+
+try:
+    import gromet
+except ModuleNotFoundError:
+    # For tests
+    import scripts.gromet.gromet_v1.gromet as gromet
```

Comment:
Given that we set `export PYTHONPATH=$PYTHONPATH:`pwd`/automates/scripts/gromet/gromet_v1` above before running the test, it's a bit confusing that this is also necessary.